README.md
## 购物app通用接口 flutter 版本
仅供学习使用,可以直接运行例子

[![pub](https://badgen.net/pub/v/dd_taoke_sdk)](https://pub.dev/packages/dd_taoke_sdk)


# 新版本
* 新增京东9块9包邮列表
* 新增京东商品详情
* 新增京东大牌折扣列表
* 新增京东实时榜单列表
* 新增京粉推荐商品查询
* 新增京东关键字搜索

![预览](https://static.saintic.com/picbed/huang/2021/09/23/1632406272625.png)

# 1.初始化
在main函数中添加如下代码
`debug` : true - 是否在控制台打印服务器返回数据 , false 隐藏

```dart
 DdTaokeUtil.instance.init('https://itbug.shop', '443',proxy: '',debug:false);
```

# 2.使用

```dart

void main() {
  // final proxy = '192.168.199.68:2333';
  DdTaokeUtil.instance.init('http://localhost', '80', proxy: '', onStart: (dio) {
    (dio.httpClientAdapter as DefaultHttpClientAdapter).onHttpClientCreate = (client) {
      client.badCertificateCallback = (cert, host, port) {
        return true;
      };
    };
  },debug: true); // 我测试用的
  // DdTaokeUtil.instance.init('http://itbug.shop', '80', proxy: ''); // 服务器的接口
  runApp(GetMaterialApp(
    debugShowCheckedModeBanner: false,
    home: MyApp(),
  ));
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('接口'),
        actions: [
          IconButton(
              icon: Icon(Icons.list),
              onPressed: () {
                Get.to(() => HotDayPage());
              })
        ],
      ),
      body: SingleChildScrollView(
        child: Wrap(
          spacing: 12,
          runSpacing: 12,
          children: [
            MyButton('超级分类', onTap: () async {
              final resut = await DdTaokeSdk.instance.getCategorys();
              toJsonView(resut);
            }),
            MyButton('轮播图', onTap: () async {
              final resut = await DdTaokeSdk.instance.getCarousel();
              toJsonView(resut);
            }),
            MyButton('品牌列表', onTap: () async {
              final result = await DdTaokeSdk.instance.getBrandList(param: BrandListParam(cid: '2', pageId: '1', pageSize: '20'));

              toJsonView(result);
            }),
            MyButton('商品列表', onTap: () async {
              final result = await DdTaokeSdk.instance.getProducts(param: ProductListParam(pageId: '1'));
              toJsonView(result);
            }),
            MyButton('商品详情', onTap: () async {
              final result = await DdTaokeSdk.instance.getProductDetail(param: ProductDetailParam(id: '32731926'));
              toJsonView(result);
            }),
            MyButton('获取品牌商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getBrandDetail(param: BrandProductParam(brandId: '7951745', pageSize: '20', pageId: '1'));
              toJsonView(result);
            }),
            MyButton('获取详情页面所需数据', onTap: () async {
              final result = await DdTaokeSdk.instance.getDetailBaseData(productId: '32448990');
              toJsonView(result);
            }),
            MyButton('高效转链', onTap: () async {
              await Get.dialog<String>(InputDialog(
                title: '请输入淘宝商品id',
              )).then((value) async {
                if (!GetUtils.isNullOrBlank(value)) {
                  final result = await DdTaokeSdk.instance.getCouponsDetail(taobaoGoodsId: value);
                  toJsonView(result);
                }
              });
            }),
            MyButton('高佣精选商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getHighCommissionProducts(param: HighCommissionParam(pageId: '1', pageSize: '20'));
              toJsonView(result);
            }),
            MyButton('获取商品的推广素材数据', onTap: () async {
              final result = await DdTaokeSdk.instance.getProductMaterial(productId: '32448990');
              toJsonView(result);
            }),
            MyButton('获取热搜榜', onTap: () async {
              final result = await DdTaokeSdk.instance.getHotSearchWorlds();
              toJsonView(result);
            }),
            MyButton('获取线报列表', onTap: () async {
              final result = await DdTaokeSdk.instance.getSpeiderList(param: SpeiderParam(pageId: '2', pageSize: '10', topic: '1'));
              toJsonView(result);
            }),
            MyButton('超级搜索', onTap: () async {
              final result = await DdTaokeSdk.instance.superSearch(
                  param: SuperSearchParam(
                keyWords: '辣条',
                pageSize: '1',
                type: '0',
                pageId: '1',
              ));
              toJsonView(result);
            }),
            MyButton('淘宝官方活动(一元购)', onTap: () async {
              final result = await DdTaokeSdk.instance.getTaobaoOnepriceProducts(param: TaobaoOnePriceParam(pageId: '1'));
              toJsonView(result);
            }),
            MyButton('朋友圈文案商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getWechat(param: WechatParam(pageId: '1', pageSize: '10'));
              toJsonView(result);
            }),
            MyButton('获取榜单商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getTopProducts(param: TopParam(pageId: '1', rankType: '1'));
              toJsonView(result);
            }),
            MyButton('九块九包邮', onTap: () async {
              final result = await DdTaokeSdk.instance.getNineNineProducts(param: NineNineParam(pageId: '1', nineCid: '-1', pageSize: '20'));
              toJsonView(result);
            }),
            MyButton('获取评论(暂无返回数据)', onTap: () async {
              final result = await DdTaokeSdk.instance.getProductComments(param: CommentParam(id: '32731926'));
              toJsonView(result);
            }),
            MyButton('店铺转链', onTap: () async {
              final result = await DdTaokeSdk.instance.shopConvert(param: ShopConvertParam(sellerId: '', shopName: null, pid: null));
              toJsonView(result);
            }),
            MyButton('获取细分类目商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getSubdivisionProducts(subdivisionId: '249');
              toJsonView(result);
            }),
            MyButton('折上折', onTap: () async {
              final result = await DdTaokeSdk.instance.getDiscountTwoProduct(param: DiscountTwoParam(pageId: '1', pageSize: '10', sort: DdSort.defaultSort));
              toJsonView(result);
            }),
            MyButton('每日半价', onTap: () async {
              final result = await DdTaokeSdk.instance.getHalfdayProducts();
              toJsonView(result);
            }),
            MyButton('获取商品历史价格', onTap: () async {
              final result = await DdTaokeSdk.instance.getProductHistoryPrice(productId: '32731926');
              toJsonView(result);
            }),
            MyButton('直播好货', onTap: () async {
              final result = await DdTaokeSdk.instance.getLiveDataProducts();
              toJsonView(result);
            }),
            MyButton('每日爆品', onTap: () async {
              final result = await DdTaokeSdk.instance.getHotDayProduct(param: HotdayParam(pageId: '1', pageSize: '10'));
              toJsonView(result);
            }),
            MyButton('咚咚抢', onTap: () async {
              final result = await DdTaokeSdk.instance.getDdq();
              toJsonView(result);
            }),
            MyButton('获取线报(时间段抢购)', onTap: () async {
              final result = await DdTaokeSdk.instance.getSpeiderListWithTime(param: SpeiderParam());
              toJsonView(result);
            }),
            MyButton('活动转链', onTap: () async {
              final activityId = await Get.dialog<String>(InputDialog(
                title: '请输入活动id',
              ));
              if (activityId != null) {
                final result = await DdTaokeSdk.instance.getActivityLink(ActivityLinkParam(promotionSceneId: activityId));
                toJsonView(result);
              }
            }),
            MyButton('搜索建议', onTap: () async {
              final resul = await DdTaokeSdk.instance.getSuggest();
              print(resul.length);
            }),
            MyButton('登录', onTap: () async {
              final result = DdTaokeUtil.instance.post('/api/login', data: {'username': 'admin', 'password': '123456'});
              toJsonView(result);
            }),
            MyButton('判断浏览器版本', onTap: () async {
              if (kIsWeb) {}
            }),
            MyButton('京东9块9', onTap: () async {
             final products =  await DdTaokeSdk.instance.jdNinesList(5, 20, 0);
             toJsonView(products);
            }),
            MyButton('京东产品详情', onTap: () async {
              final products =  await DdTaokeSdk.instance.jdDetail('10327875287');
              toJsonView(products);
            }),
            MyButton('京东实时榜单', onTap: () async {
              final products =  await DdTaokeSdk.instance.jdPhb(20, 20);
              toJsonView(products);
            }),
            MyButton('京东大牌折扣商品', onTap: () async {
              final products =  await DdTaokeSdk.instance.jdDpzk(1, 20);
              toJsonView(products);
            }),
            MyButton('用户注册测试', onTap: (){
              PublicApi.req.register('test', '111', '头像url');
            }),
            MyButton('登录测试', onTap: (){
              PublicApi.req.login('test', '11122',tokenHandle: (token){
                print('登录获取的token是:$token');
              },loginFail: (msg){
                print('登录失败:$msg');
              });
            })
            ,
            MyButton('获取系统预设头像', onTap: ()async{
             final result = await PublicApi.req.getAvaPics();
             toJsonView(result);
            }),
            MyButton('创建游戏房间', onTap: (){
              PublicApi.req.createRoom(46, '梁典典进');
            }),
            MyButton('获取全部的游戏房间', onTap: () async {
              final result = await PublicApi.req.getAllRoom();
              print('获取到房间数量:${result.length}');
            }),
            MyButton('获取当前在线总人数', onTap: (){
              PublicApi.req.getInlineUserCount();
            })
          ],
        ),
      ),
    );
  }
}


```



CHANGELOG.md
#1.6.1
* 修复iOS找不到头文件的错误

##1.6.0
* 修复一处包名错误

##1.5.8
* 修复iOS下的错误

##1.5.7
* 新增京东的分类查询接口
```dart
DdTaokeSdk.instance.getJdCategory(parentId: '0'); 
```

##1.5.6
* 新增拼多多的分类查询接口
```dart
DdTaokeSdk.instance.getPddCategory('0'); 
```

##1.5.5
* 修复若干bug

##1.5.4
* 新增京粉推荐商品接口
* 新增京东商品查询接口
```dart
final jdApi = JdApi.instance; // 使用
```

##1.5.2
* 新增请求参数的打印

##1.5.1
* 新增进入房间的接口

##1.5.0
* 新增删除房间的接口

##1.4.9
* 新增获取在线人数的接口

##1.4.8
* 新增创建房间的接口
* 新增获取所有房间的接口

##1.4.6
* 升级get依赖到v4.3.8
* 新增用户注册接口
* 新增用户登录接口
* 新增用户jwt获取用户信息接口

##1.4.3
* 新增京东9块9包邮列表
* 新增京东商品详情
* 新增京东大牌折扣列表
* 新增京东实时榜单列表

##1.3.9
* 修复品牌产品对象部分出现为null的错误

##1.3.8
* 修复品牌产品对象部分出现为null的错误

##1.3.7
* bug fixed

##1.3.6
* bug fixed

##1.3.5
* 插件引入方式变更 (web)
* 新增判断微信浏览器Api
```dart
  // 判断是否微信浏览器
  final isWeChat =  await FlutterDdDataokeSdk.isWebChatBrowser;
  print(isWeChat); // true 或者 false 
```

##1.3.3
* 添加判断是否为微信浏览器的api

#1.3.2 
* 移除1.3.1版本更新的内容

##1.3.1
* 添加web适配器
```dart
    if(kIsWeb){
      var adapter = BrowserHttpClientAdapter();
      adapter.withCredentials = true;
      _dio.httpClientAdapter = adapter;
    }
```

##1.3.0
* bug 修复

##1.2.8
* 修复bug

##1.2.7
* 修复bug

##1.2.6
* 新增产品列表关闭请求

##1.2.5
* some bugs fixed

##1.2.1
* 新增map data 数据处理

##1.2.0
* 新增是否微信浏览器判断

##1.1.9
* 新增debug模式

##1.1.8
* 新增朋友圈文案

##1.1.7
* bug修复

##1.1.6
* bug 修复

##1.1.5
* bug fixed

##1.1.4
* bug fixed

##1.1.3
* 暴露方法

##1.1.2
*bug fixed

##1.1.1
* bug fixed

##1.1.0
* bug fixed

## 1.0.9
*修复post错误

##1.0.8
* bug修复

##1.0.7
* bug修复

##1.0.6
* 新增post方法

##1.0.5
* 新增获取搜索建议接口

## 1.0.2
* 新增活动转链接口

## 1.0.1
* bug修复

## 1.0.0
* 升级到空安全版本

## 0.2.5

* 升级线报接口

## 0.2.4
* 升级线报接口

## 0.2.3
* 升级轮播图接口

## 0.2.2 
* 封装了一个通用刷新下拉加载的页面 `BasePage`
* 修改了部分接口实现

## 0.2.0
* 新增咚咚抢接口
* 新增每日爆品接口
* 新增直播好货接口
* 新增获取商品的历史价格接口

## 0.1.6

* 新增每日半价接口
* 新增折上折接口
* 新增细分类目商品接口
* 新增店铺转链接口
* 新增获取商品评论接口


## 0.1.1

* 新增九块九包邮接口
* 新增获取榜单商品接口
* 新增获取朋友圈文案接口
* 新增官方活动 (淘宝一元购)接口

## 0.0.8

* 新增商品精推素材接口
* 新增热搜榜接口
* 新增获取获取线报接口


## 0.0.5 
    
* 新增高佣精选接口
* 新增获取轮播图接口
* 新增商品详情接口
* 新增高薪转链接口

## 0.0.1

* TODO: Describe initial release.


example/lib/main.dart
import 'package:dd_taoke_sdk/constant/sort.dart';
import 'package:dd_taoke_sdk/dd_taoke_sdk.dart';
import 'package:dd_taoke_sdk/jd_api.dart';
import 'package:dd_taoke_sdk/model/jd/jd_req_param.dart';
import 'package:dd_taoke_sdk/network/util.dart';
import 'package:dd_taoke_sdk/params/activity_link_param.dart';
import 'package:dd_taoke_sdk/params/brand_param.dart';
import 'package:dd_taoke_sdk/params/brand_product_param.dart';
import 'package:dd_taoke_sdk/params/comment_param.dart';
import 'package:dd_taoke_sdk/params/discount_two_param.dart';
import 'package:dd_taoke_sdk/params/high_commission_param.dart';
import 'package:dd_taoke_sdk/params/hotday_param.dart';
import 'package:dd_taoke_sdk/params/nine_nine_param.dart';
import 'package:dd_taoke_sdk/params/product_detail_param.dart';
import 'package:dd_taoke_sdk/params/product_list_param.dart';
import 'package:dd_taoke_sdk/params/shop_convert_param.dart';
import 'package:dd_taoke_sdk/params/speider_param.dart';
import 'package:dd_taoke_sdk/params/super_search_param.dart';
import 'package:dd_taoke_sdk/params/taobao_oneprice_param.dart';
import 'package:dd_taoke_sdk/params/top_param.dart';
import 'package:dd_taoke_sdk/params/wechat_param.dart';
import 'package:dd_taoke_sdk/public_api.dart';
import 'component/buttom.dart';
import 'component/input_model.dart';
import 'test/hotday_page.dart';
import 'package:dio/adapter.dart';
import 'package:flutter/material.dart';
import 'package:get/get.dart';

import 'component/json_result_page.dart';

void main() {
  // final proxy = '192.168.199.68:2333';
  DdTaokeUtil.instance.init('http://localhost', '80', proxy: '', onStart: (dio) {
    (dio.httpClientAdapter as DefaultHttpClientAdapter).onHttpClientCreate = (client) {
      client.badCertificateCallback = (cert, host, port) {
        return true;
      };
    };
  },debug: false,printParams: true); // 我测试用的
  // DdTaokeUtil.instance.init('http://itbug.shop', '80', proxy: ''); // 服务器的接口
  runApp(GetMaterialApp(
    debugShowCheckedModeBanner: false,
    home: MyApp(),
  ));
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('接口'),
        actions: [
          IconButton(
              icon: Icon(Icons.list),
              onPressed: () {
                Get.to(() => HotDayPage());
              })
        ],
      ),
      body: SingleChildScrollView(
        child: Wrap(
          spacing: 12,
          runSpacing: 12,
          children: [
            MyButton('超级分类', onTap: () async {
              final resut = await DdTaokeSdk.instance.getCategorys();
              toJsonView(resut);
            }),
            MyButton('轮播图', onTap: () async {
              final resut = await DdTaokeSdk.instance.getCarousel();
              toJsonView(resut);
            }),
            MyButton('品牌列表', onTap: () async {
              final result = await DdTaokeSdk.instance.getBrandList(param: BrandListParam(cid: '2', pageId: '1', pageSize: '20'));

              toJsonView(result);
            }),
            MyButton('商品列表', onTap: () async {
              final result = await DdTaokeSdk.instance.getProducts(param: ProductListParam(pageId: '1'));
              toJsonView(result);
            }),
            MyButton('商品详情', onTap: () async {
              final result = await DdTaokeSdk.instance.getProductDetail(param: ProductDetailParam(id: '35685974'));
              toJsonView(result);
            }),
            MyButton('获取品牌商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getBrandDetail(param: BrandProductParam(brandId: '7951745', pageSize: '20', pageId: '1'));
              toJsonView(result);
            }),
            MyButton('获取详情页面所需数据', onTap: () async {
              final result = await DdTaokeSdk.instance.getDetailBaseData(productId: '35685974');
              toJsonView(result);
            }),
            MyButton('高效转链', onTap: () async {
              await Get.dialog<String>(InputDialog(
                title: '请输入淘宝商品id',
              )).then((value) async {
                if (!GetUtils.isNullOrBlank(value)) {
                  final result = await DdTaokeSdk.instance.getCouponsDetail(taobaoGoodsId: value);
                  toJsonView(result);
                }
              });
            }),
            MyButton('高佣精选商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getHighCommissionProducts(param: HighCommissionParam(pageId: '1', pageSize: '20'));
              toJsonView(result);
            }),
            MyButton('获取商品的推广素材数据', onTap: () async {
              final result = await DdTaokeSdk.instance.getProductMaterial(productId: '32448990');
              toJsonView(result);
            }),
            MyButton('获取热搜榜', onTap: () async {
              final result = await DdTaokeSdk.instance.getHotSearchWorlds();
              toJsonView(result);
            }),
            MyButton('获取线报列表', onTap: () async {
              final result = await DdTaokeSdk.instance.getSpeiderList(param: SpeiderParam(pageId: '2', pageSize: '10', topic: '1'));
              toJsonView(result);
            }),
            MyButton('超级搜索', onTap: () async {
              final result = await DdTaokeSdk.instance.superSearch(
                  param: SuperSearchParam(
                keyWords: '辣条',
                pageSize: '1',
                type: '0',
                pageId: '1',
              ));
              toJsonView(result);
            }),
            MyButton('淘宝官方活动(一元购)', onTap: () async {
              final result = await DdTaokeSdk.instance.getTaobaoOnepriceProducts(param: TaobaoOnePriceParam(pageId: '1'));
              toJsonView(result);
            }),
            MyButton('朋友圈文案商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getWechat(param: WechatParam(pageId: '1', pageSize: '10'));
              toJsonView(result);
            }),
            MyButton('获取榜单商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getTopProducts(param: TopParam(pageId: '1', rankType: '1'));
              toJsonView(result);
            }),
            MyButton('九块九包邮', onTap: () async {
              final result = await DdTaokeSdk.instance.getNineNineProducts(param: NineNineParam(pageId: '1', nineCid: '-1', pageSize: '20'));
              toJsonView(result);
            }),
            MyButton('获取评论(暂无返回数据)', onTap: () async {
              final result = await DdTaokeSdk.instance.getProductComments(param: CommentParam(id: '32731926'));
              toJsonView(result);
            }),
            MyButton('店铺转链', onTap: () async {
              final result = await DdTaokeSdk.instance.shopConvert(param: ShopConvertParam(sellerId: '', shopName: null, pid: null));
              toJsonView(result);
            }),
            MyButton('获取细分类目商品', onTap: () async {
              final result = await DdTaokeSdk.instance.getSubdivisionProducts(subdivisionId: '249');
              toJsonView(result);
            }),
            MyButton('折上折', onTap: () async {
              final result = await DdTaokeSdk.instance.getDiscountTwoProduct(param: DiscountTwoParam(pageId: '1', pageSize: '10', sort: DdSort.defaultSort));
              toJsonView(result);
            }),
            MyButton('每日半价', onTap: () async {
              final result = await DdTaokeSdk.instance.getHalfdayProducts();
              toJsonView(result);
            }),
            MyButton('获取商品历史价格', onTap: () async {
              final result = await DdTaokeSdk.instance.getProductHistoryPrice(productId: '32731926');
              toJsonView(result);
            }),
            MyButton('直播好货', onTap: () async {
              final result = await DdTaokeSdk.instance.getLiveDataProducts();
              toJsonView(result);
            }),
            MyButton('每日爆品', onTap: () async {
              final result = await DdTaokeSdk.instance.getHotDayProduct(param: HotdayParam(pageId: '1', pageSize: '10'));
              toJsonView(result);
            }),
            MyButton('咚咚抢', onTap: () async {
              final result = await DdTaokeSdk.instance.getDdq();
              toJsonView(result);
            }),
            MyButton('获取线报(时间段抢购)', onTap: () async {
              final result = await DdTaokeSdk.instance.getSpeiderListWithTime(param: SpeiderParam());
              toJsonView(result);
            }),
            MyButton('活动转链', onTap: () async {
              final activityId = await Get.dialog<String>(InputDialog(
                title: '请输入活动id',
              ));
              if (activityId != null) {
                final result = await DdTaokeSdk.instance.getActivityLink(ActivityLinkParam(promotionSceneId: activityId));
                toJsonView(result);
              }
            }),
            MyButton('搜索建议', onTap: () async {
              final resul = await DdTaokeSdk.instance.getSuggest();
              toJsonView(resul);
            }),
            MyButton('京东9块9', onTap: () async {
             final products =  await DdTaokeSdk.instance.jdNinesList(5, 20, 0);
             toJsonView(products);
            }),
            MyButton('京东产品详情', onTap: () async {
              final products =  await DdTaokeSdk.instance.jdDetail('10327875287');
              toJsonView(products);
            }),
            MyButton('京东实时榜单', onTap: () async {
              final products =  await DdTaokeSdk.instance.jdPhb(20, 20);
              toJsonView(products);
            }),
            MyButton('京东大牌折扣商品', onTap: () async {
              final products =  await DdTaokeSdk.instance.jdDpzk(1, 20);
              toJsonView(products);
            }),
            MyButton('用户注册测试', onTap: (){
              PublicApi.req.register('diandian', '1234567', 'https://static.saintic.com/picbed/huang/2021/08/17/1629169985486.jpg',otherDataHandle: (data){
                print(data);
              },apiError: (code,msg,data){
                print(code);
                print(msg);
                print(data);
              }).then((value) {
                if(value){
                  print('注册成功');
                }
              });
            }),
            MyButton('登录测试', onTap: (){
              PublicApi.req.login('diandian', '1234567',tokenHandle: (token){
                print('登录获取的token是:$token');
              },loginFail: (msg){
                print('登录失败:$msg');
              });
            })
            ,
            MyButton('获取系统预设头像', onTap: ()async{
             final result = await PublicApi.req.getAvaPics();
             toJsonView(result);
            }),
            MyButton('创建游戏房间', onTap: (){
              PublicApi.req.createRoom(46, '梁典典进');
            }),
            MyButton('获取全部的游戏房间', onTap: () async {
              final result = await PublicApi.req.getAllRoom();
              print('获取到房间数量:${result.length}');
            }),
            MyButton('获取当前在线总人数', onTap: (){
              PublicApi.req.getInlineUserCount();
            }),
            MyButton('获取京东商品', onTap: ()async{

             final result= await JdApi.instance.getProducts(JdReqParam(eliteId: 2));
             toJsonView(result);
              
            }),
            MyButton('京东商品查询', onTap: ()async{

              final result= await JdApi.instance.searchProducts(error: (c,m,d){
                print(c);
                print(m);
                print(d);
              },params: {'keyword':'辣条'});
              toJsonView(result);

            }),

            MyButton('拼多多的分类查询',onTap: ()async {
              final result = await DdTaokeSdk.instance.getPddCategory('0');
              toJsonView(result);
            },),
            MyButton('京东的分类查询',onTap: ()async {
              final result = await DdTaokeSdk.instance.getJdCategory(parentId: '0');
              toJsonView(result);
            },)
          ],
        ),
      ),
    );
  }
}
